### PR TITLE
Alluxio

### PR DESCRIPTION
**Pull request by apc999**
### What is this PR for?

Zeppelin provides an Alluxio interpreter compiled with Alluxio client v1.0.0, which was released about two years ago. Alluxio 1.0.0 client will not work with Alluxio 1.4 or later. So in order to use Zeppelin with latest Alluxio, we need to increment the version.

This JIRA aims to bump the version of Alluxio from 1.0.0 to 1.7.1 (the latest release in March 2018)

### What type of PR is it?
[Improvement]

### What is the Jira issue?
[ZEPPELIN-3373](https://issues.apache.org/jira/browse/ZEPPELIN-3373)

### How should this be tested?
Manual test:
- Build Zeppelin with this patch
- Launch Zeppelin Alluxio interpreter and test the interpreter. 

### Screenshots (if appropriate)
[Screenshot](https://issues.apache.org/jira/secure/attachment/12916912/Screen%20Shot%202018-03-27%20at%205.32.36%20PM.png)

### Questions:
* Does the licenses files need update? n/a
* Is there breaking changes for older versions? n/a
* Does this needs documentation? [docs](https://zeppelin.apache.org/docs/latest/interpreter/alluxio.html) needs to be updated

### CODE REVIEW

**Comment by apc999** [comment]
anyone to review or comment this PR?

File: alluxio/src/test/java/org/apache/zeppelin/alluxio/AlluxioInterpreterTest.java [hunk 1]
```diff
@@ -52,12 +54,50 @@
 import org.apache.zeppelin.interpreter.InterpreterResult.Code;
 import org.apache.zeppelin.interpreter.thrift.InterpreterCompletion;
 
+@Ignore("https://alluxio.atlassian.net/browse/ALLUXIO-3142")
```
Comment (felixcheung) [inline alluxio/src/test/java/org/apache/zeppelin/alluxio/AlluxioInterpreterTest.java]: this disables the whole test?

File: alluxio/src/test/java/org/apache/zeppelin/alluxio/AlluxioInterpreterTest.java [hunk 2]
```diff
@@ -34,15 +35,16 @@
 
 import alluxio.AlluxioURI;
 import alluxio.Constants;
-import alluxio.client.FileSystemTestUtils;
```
Comment (felixcheung) [inline alluxio/src/test/java/org/apache/zeppelin/alluxio/AlluxioInterpreterTest.java]: so no more FileSystemTestUtils we can use?

File: alluxio/src/main/java/org/apache/zeppelin/alluxio/AlluxioInterpreter.java [hunk 3]
```diff
@@ -99,26 +95,22 @@ public InterpreterResult interpret(String st, InterpreterContext context) {
...
     System.setOut(ps);
-    
+
     for (String command : commands) {
       int commandResult = 1;
       String[] args = splitAndRemoveEmpty(command, " ");
-      if (args.length > 0 && args[0].equals("help")) {
-        System.out.println(getCommandList());
-      } else {
-        commandResult = fs.run(args);
```
Comment (felixcheung) [inline alluxio/src/main/java/org/apache/zeppelin/alluxio/AlluxioInterpreter.java]: is there a replacement of the command list for help?

File: alluxio/pom.xml [hunk 4]
```diff
@@ -106,7 +106,15 @@
 
         <dependency>
             <groupId>org.alluxio</groupId>
-            <artifactId>alluxio-core-server</artifactId>
+            <artifactId>alluxio-core-common</artifactId>
```
Comment (felixcheung) [inline alluxio/pom.xml]: this list needs to be updated https://github.com/apache/zeppelin/blob/master/zeppelin-distribution/src/bin_license/LICENSE#L103

if we are not using core-server for example, plus new version numbers

**Comment by apc999** [comment]
@felixcheung hi I am just back from a trip and I will work on this now

**Comment by felixcheung** [comment]
ping

**Comment by jongyoul** [comment]
ping

**Comment by antonkulaga** [comment]
@apc999 could you please update the client to 1.8.0 ?

**Comment by felixcheung** [comment]
ping?

**Comment by rblaine95** [comment]
Any plans to update to Alluxio Shell 2.0.0?

**Comment by jongyoul** [comment]
We are currently providing Alluxio 2.5 so I'll close this PR. Thank you for your interest.

**Pull request closed without merging**